### DISCUSSION TITLE
MrMapi recipent table

### DISCUSSION THREAD

**Discussion by chaosblog** [microsoft/mfcmapi#503] [Q&A] [answered]
Is it possible to extract the recipient table of an .msg-file with the help of MrMapi?

I need a way to get the sender and all the recipient mail addresses to process them in a powershell script.

Thanks for your help.

**Comment by stephenegriffin** [comment, marked as answer]
You can convert the msg to xml and then process it however you like:
mrmapi -xml -input <path_to_msg> -output <path_to_xml>
